test/visual/story-imports.js

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
I'd target long term being able to build this file from content, but I've messed with this VRT thing a little too much and I didn't want to add any extra responsibilities to getting this change in at this time. Hopefully, it's already more clear than the stories array we were maintaining previously.